client/src/components/UserCard.tsx

```diff
@@ -49,27 +82,36 @@ export const UserCard: React.FC<IUser> = (user) => {
       <CardText>
         <CardHeader>
           <CardName>
-            {user.gitHubConnected
-              ? user.gitHub.json.name || user.gitHub.json.login
-              : user.discord.username}
+            {username(user)}
           </CardName>
           <CardLocation>
-            {user.gitHubConnected ? user.gitHub.json.location : null}
+            {location(user)}
           </CardLocation>
         </CardHeader>
-        <CardBio>{user.gitHubConnected ? user.gitHub.json.bio : null}</CardBio>
+        <CardBio>
+          {bio(user)}
+        </CardBio>
         <CardMeta>
-          {user.gitHub && user.gitHub.json && 
-            <CardMetaFeature href={ "https://www.twitter.com/" + user.gitHub.json?.twitter_username} target="_blank"> 
```

Comment:
Hey, @theodorusandi, I figured out #119 here. For some reason, I had decided to grab the Twitter username data from GitHub... instead of Twitter. So you likely did not have your Twitter account linked on GitHub. This should fix the issue!